deploy/kubernetes/driver/kubernetes/overlays/stage/controller-server-images.yaml

```diff
@@ -12,8 +12,8 @@ spec:
           image: icr.io/ext/sig-storage/livenessprobe:v2.3.0
         - name: iks-vpc-file-driver
           imagePullPolicy: Always
-          image: icr.io/obs/storage/ibm-vpc-file-csi-driver:v1.0.0-beta1
+          image: icr.io/obs/storage/ibm-vpc-file-csi-driver:v1.0.1-beta
         - name: storage-secret-sidecar
-          image: icr.io/obs/armada-storage-secret:v1.1.4
+          image: icr.io/obs/armada-storage-secret:v1.1.12
         - name: csi-resizer
-          image: quay.io/k8scsi/csi-resizer:v1.2.0
+          image: icr.io/ext/sig-storage/csi-resizer:v1.2.0
```

Comment:
In community I see new versions of resizer and other sidecars. Can we consider upgrading .